main/build.gradle

```diff
@@ -58,6 +59,18 @@ repositories {
     mavenCentral()
 }
 
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            groupId = 'org.mobilitydata'
```

Comment:
`'org.mobilitydata'` is repeated multiple times in this file. It would be safer to define a variable and reference that throughout.